* grobian@gentoo.org: add needed libraries to satisfy all dependencies

--- Makefile
+++ Makefile
@@ -1,4 +1,4 @@
-LIBS=$(shell pkg-config --libs poppler) -lXm
+LIBS=$(shell pkg-config --libs poppler) -lXm -lXt -lX11 -lfontconfig
 POPPLERBASE=$(strip $(shell pkg-config --cflags poppler))
 INCLUDES=$(POPPLERBASE) $(POPPLERBASE)/goo $(POPPLERBASE)/splash
 
